assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
```

Comment:
Structurally it's still an HTML element (a link), just with a variant styling. So I think it's fine to include with elements. I think this will add more confusion by separating it out from the default links. 

We also have `components` for more complex things made up of more than one elements.